test/run

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+# This is the expected entry point for Cockpit CI; will be called without
+# arguments but with an appropriate $TEST_OS, and optionally $TEST_SCENARIO
+
+make vm
+
+if [ -n "$TEST_SCENARIO" ]; then
+  test/check-cloud TestCloud.test_$TEST_SCENARIO
```

Comment:
Can $TEST_SCENARIO actually be defined as Class.test_Method? Then we can move all of the runCliTest methods in one file and be able to start whichever scenario we want. Sort of treating the new Python unittest files like a substitute for Makefile.

Reply:
I think this is going a bit far. The test runners won't run all those jobs in parallel anyway. Are you ok with going forward like this and changing it if we notice problems?

Reply:
+1 for merging as-is and I can tweak the rest later.